pkg/storage/snapshot/snapshot_test.go

```diff
@@ -748,7 +747,7 @@ var _ = Describe("Snapshot controlleer", func() {
 			It("should (partial) lock source if running", func() {
 				vmSnapshot := createVMSnapshotInProgress()
 				vm := createVM()
-				vm.Spec.Running = &t
+				vm.Spec.RunStrategy = pointer.P(v1.RunStrategyAlways)
```

Comment:
aren't `f` and `t` used anymore? I don't see here their removal

Reply:
this is used for ReadyToUse = &t. But I dont mind replacing that with the pointer like I did in other file

Reply:
@alicefr added another commit to clean all pointers to t and f